src/rct/printing.py

```diff
@@ -32,6 +32,8 @@
 def xstr(value):
     if value is None:
         return ''
+    elif isinstance(value, list):
+        return ", ".join([xstr(val) for val in value])
```

Comment:
I believe we should keep the standard way of showing multi-value fields, listed with a new line between them, just like other fields are, e.g. provided products are listed like this:
```
Provides:            SP Additional bits
                     SP Addon 2 bits
                     SP Addon 3 bits
```
So the addons/roles should also be listed like this:
```
Add-ons:             ADDON1
                     ADDON2
                     ADDON3
```
in both cases (`--consumed` and `--available`).